integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/validation/InstanceOfValidator.java

```diff
@@ -0,0 +1,20 @@
+package no.difi.meldingsutveksling.validation;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+@Slf4j
+public class InstanceOfValidator implements ConstraintValidator<InstanceOf, Object> {
+
+    private Class<?> expectedClass;
+
+    public void initialize(InstanceOf constraint) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)